pom.xml

```diff
@@ -263,6 +263,7 @@
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>${maven-surefire-plugin.version}</version>
                     <configuration>
+                        <failIfNoSpecifiedTests>false</failIfNoSpecifiedTests>
```

Comment:
I think you added this to address "Due to the way the surefire plugin works when multiple projects are present, if you run with the -Dtest option from the top level, mvn will report a BUILD FAILURE regardless of the test results. To avoid this, (a) run from the relevant subdirectory such as jgrapht-core or (b) use the -DfailIfNoTests=false flag."

But the negative effect is that if someone has a typo in the specific test name they entered, they'll get a success, and assume it's passing.  So I think we shouldn't change it.

Reply:
> I think you added this to address "Due to the way the surefire plugin works when multiple projects are present, if you run with the -Dtest option from the top level, mvn will report a BUILD FAILURE regardless of the test results.

As you guessed, that was the exact reason.

> To avoid this, (a) run from the relevant subdirectory such as jgrapht-core or (b) use the -DfailIfNoTests=false flag."

I think this should be documented in Wiki.

> But the negative effect is that if someone has a typo in the specific test name they entered, they'll get a success, and assume it's passing. So I think we shouldn't change it.

Fair point. Will revert it shortly.